const.go

```diff
@@ -69,10 +72,17 @@ const (
 	RLimitDrop = 1
 	RLimitWait = 2
 
+	FeatureStreams        = uint64(0)
+	FeatureTransactions   = uint64(1)
+	FeatureErrorExtension = uint64(2)
+	FeatureWatchers       = uint64(3)
```

Comment:
Seems like a good idea to make a separate enumeration type:

https://github.com/tarantool/go-tarantool/blob/48cf0c75b3869e54fc47881f0a1323612c3694fe/stream.go#L9-L23

+

In general, you could extract some of the pull request code in a separate file (`features.go` ?).

Reply:
Reworked the code